src/main/java/com/hubspot/jinjava/lib/exptest/IsSequenceExpTest.java

```diff
@@ -25,9 +27,6 @@ public String getName() {
 
   @Override
   public boolean evaluate(Object var, JinjavaInterpreter interpreter, Object... args) {
-    return (
-      var != null &&
-      (var.getClass().isArray() || Iterable.class.isAssignableFrom(var.getClass()))
-    );
+    return isIterable(var);
```

Comment:
It is questionable to define iterables and sequences as the same thing, but this follows the Jinja spec.